### PR TITLE
Upgrade to node 8.5.0

### PR DESCRIPTION
**Pull request by SimenB**
https://github.com/nodejs/docker-node/pull/522

https://nodejs.org/en/blog/release/v8.5.0/

### CODE REVIEW

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index ca0a2ba..d18da6e 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,25 @@
-node:alpine @ amd64
+node:8.5 @ amd64
+node:8.5 @ arm32v7
+node:8.5 @ arm64v8
+node:8.5 @ ppc64le
+node:8.5 @ s390x
+node:8.5-alpine @ amd64
+node:8.5-onbuild @ amd64
+node:8.5-onbuild @ arm32v7
+node:8.5-onbuild @ arm64v8
+node:8.5-onbuild @ ppc64le
+node:8.5-onbuild @ s390x
+node:8.5-slim @ amd64
+node:8.5-slim @ arm32v7
+node:8.5-slim @ arm64v8
+node:8.5-slim @ ppc64le
+node:8.5-slim @ s390x
+node:8.5-stretch @ amd64
+node:8.5-stretch @ arm32v7
+node:8.5-stretch @ arm64v8
+node:8.5-stretch @ ppc64le
+node:8.5-stretch @ s390x
+node:8.5-wheezy @ amd64
 node:argon @ amd64
 node:argon @ arm32v7
 node:argon @ arm64v8
@@ -39,24 +60,3 @@ node:boron-stretch @ arm64v8
 node:boron-stretch @ ppc64le
 node:boron-stretch @ s390x
 node:boron-wheezy @ amd64
-node:latest @ amd64
-node:latest @ arm32v7
-node:latest @ arm64v8
-node:latest @ ppc64le
-node:latest @ s390x
-node:onbuild @ amd64
-node:onbuild @ arm32v7
-node:onbuild @ arm64v8
-node:onbuild @ ppc64le
-node:onbuild @ s390x
-node:slim @ amd64
-node:slim @ arm32v7
-node:slim @ arm64v8
-node:slim @ ppc64le
-node:slim @ s390x
-node:stretch @ amd64
-node:stretch @ arm32v7
-node:stretch @ arm64v8
-node:stretch @ ppc64le
-node:stretch @ s390x
-node:wheezy @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index dcc8f6c..9f5dd82 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -34,25 +34,18 @@ node:6.11.3-onbuild
 node:6.11.3-slim
 node:6.11.3-stretch
 node:6.11.3-wheezy
-node:8
-node:8-alpine
-node:8-onbuild
-node:8-slim
-node:8-stretch
-node:8-wheezy
-node:8.4
-node:8.4-alpine
-node:8.4-onbuild
-node:8.4-slim
-node:8.4-stretch
-node:8.4-wheezy
-node:8.4.0
-node:8.4.0-alpine
-node:8.4.0-onbuild
-node:8.4.0-slim
-node:8.4.0-stretch
-node:8.4.0-wheezy
-node:alpine
+node:8.5
+node:8.5-alpine
+node:8.5-onbuild
+node:8.5-slim
+node:8.5-stretch
+node:8.5-wheezy
+node:8.5.0
+node:8.5.0-alpine
+node:8.5.0-onbuild
+node:8.5.0-slim
+node:8.5.0-stretch
+node:8.5.0-wheezy
 node:argon
 node:argon-alpine
 node:argon-onbuild
@@ -65,8 +58,3 @@ node:boron-onbuild
 node:boron-slim
 node:boron-stretch
 node:boron-wheezy
-node:latest
-node:onbuild
-node:slim
-node:stretch
-node:wheezy
diff --git a/node_alpine/Dockerfile b/node_8.5-alpine/Dockerfile
similarity index 98%
rename from node_alpine/Dockerfile
rename to node_8.5-alpine/Dockerfile
index ffc205d..41cdda2 100644
--- a/node_alpine/Dockerfile
+++ b/node_8.5-alpine/Dockerfile
@@ -1,7 +1,7 @@
 FROM alpine:3.6
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
@@ -46,7 +46,7 @@ RUN addgroup -g 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
   && for key in \
diff --git a/node_onbuild/Dockerfile b/node_8.5-onbuild/Dockerfile
similarity index 93%
rename from node_onbuild/Dockerfile
rename to node_8.5-onbuild/Dockerfile
index 7bc4597..ea91895 100644
--- a/node_onbuild/Dockerfile
+++ b/node_8.5-onbuild/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:8.4.0
+FROM node:8.5.0
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
diff --git a/node_slim/Dockerfile b/node_8.5-slim/Dockerfile
similarity index 98%
rename from node_slim/Dockerfile
rename to node_8.5-slim/Dockerfile
index bcaa40b..0faf72b 100644
--- a/node_slim/Dockerfile
+++ b/node_8.5-slim/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN buildDeps='xz-utils' \
     && ARCH= && dpkgArch="$(dpkg --print-architecture)" \
@@ -45,7 +45,7 @@ RUN buildDeps='xz-utils' \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
diff --git a/node_stretch/Dockerfile b/node_8.5-stretch/Dockerfile
similarity index 98%
rename from node_stretch/Dockerfile
rename to node_8.5-stretch/Dockerfile
index 4826e8d..bcc39f3 100644
--- a/node_stretch/Dockerfile
+++ b/node_8.5-stretch/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
@@ -40,7 +40,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
diff --git a/node_wheezy/Dockerfile b/node_8.5-wheezy/Dockerfile
similarity index 98%
rename from node_wheezy/Dockerfile
rename to node_8.5-wheezy/Dockerfile
index 0e1fe45..441a7d2 100644
--- a/node_wheezy/Dockerfile
+++ b/node_8.5-wheezy/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
@@ -37,7 +37,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
diff --git a/node_latest/Dockerfile b/node_8.5/Dockerfile
similarity index 98%
rename from node_latest/Dockerfile
rename to node_8.5/Dockerfile
index 13c85fd..46851f2 100644
--- a/node_latest/Dockerfile
+++ b/node_8.5/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
@@ -40,7 +40,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
````

</details>

File: library/node [hunk 1]
```diff
@@ -3,35 +3,35 @@
 Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)
 GitRepo: https://github.com/nodejs/docker-node.git
 
-Tags: 8.4.0, 8.4, 8, latest
+Tags: 8.5.0, 8.5
```
Comment (tianon) [inline library/node]: Looks like `latest` got lost. :smile:

Comment (SimenB) [reply library/node]: Good catch, thanks!

https://github.com/nodejs/docker-node/pull/523

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index dcc8f6c..0ff4429 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -40,18 +40,18 @@ node:8-onbuild
 node:8-slim
 node:8-stretch
 node:8-wheezy
-node:8.4
-node:8.4-alpine
-node:8.4-onbuild
-node:8.4-slim
-node:8.4-stretch
-node:8.4-wheezy
-node:8.4.0
-node:8.4.0-alpine
-node:8.4.0-onbuild
-node:8.4.0-slim
-node:8.4.0-stretch
-node:8.4.0-wheezy
+node:8.5
+node:8.5-alpine
+node:8.5-onbuild
+node:8.5-slim
+node:8.5-stretch
+node:8.5-wheezy
+node:8.5.0
+node:8.5.0-alpine
+node:8.5.0-onbuild
+node:8.5.0-slim
+node:8.5.0-stretch
+node:8.5.0-wheezy
 node:alpine
 node:argon
 node:argon-alpine
diff --git a/node_alpine/Dockerfile b/node_alpine/Dockerfile
index ffc205d..41cdda2 100644
--- a/node_alpine/Dockerfile
+++ b/node_alpine/Dockerfile
@@ -1,7 +1,7 @@
 FROM alpine:3.6
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
@@ -46,7 +46,7 @@ RUN addgroup -g 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
   && for key in \
diff --git a/node_latest/Dockerfile b/node_latest/Dockerfile
index 13c85fd..46851f2 100644
--- a/node_latest/Dockerfile
+++ b/node_latest/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
@@ -40,7 +40,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
diff --git a/node_onbuild/Dockerfile b/node_onbuild/Dockerfile
index 7bc4597..ea91895 100644
--- a/node_onbuild/Dockerfile
+++ b/node_onbuild/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:8.4.0
+FROM node:8.5.0
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
diff --git a/node_slim/Dockerfile b/node_slim/Dockerfile
index bcaa40b..0faf72b 100644
--- a/node_slim/Dockerfile
+++ b/node_slim/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN buildDeps='xz-utils' \
     && ARCH= && dpkgArch="$(dpkg --print-architecture)" \
@@ -45,7 +45,7 @@ RUN buildDeps='xz-utils' \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
diff --git a/node_stretch/Dockerfile b/node_stretch/Dockerfile
index 4826e8d..bcc39f3 100644
--- a/node_stretch/Dockerfile
+++ b/node_stretch/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
@@ -40,7 +40,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
diff --git a/node_wheezy/Dockerfile b/node_wheezy/Dockerfile
index 0e1fe45..441a7d2 100644
--- a/node_wheezy/Dockerfile
+++ b/node_wheezy/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.4.0
+ENV NODE_VERSION 8.5.0
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
@@ -37,7 +37,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.27.5
+ENV YARN_VERSION 1.0.1
 
 RUN set -ex \
   && for key in \
```

</details>

**Comment by yosifkit** [comment]
Build test of #3438; fc5bb256 (`node`):

```console

$ bashbrew build node:8.5.0
Using bashbrew/cache:882e8faad053c279d87a5cb6d14c77d7fd1ee174fe5686c172c317f8ec1d4408 (node:8.5.0)
Tagging node:8.5.0
Tagging node:8.5
Tagging node:8
Tagging node:latest

$ test/run.sh node:8.5.0
testing node:8.5.0
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.5.0-alpine
Building bashbrew/cache:56b2894a1cc3a3c2d731afdeafea6236be44985c6573c44bd9ba044a886ef606 (node:8.5.0-alpine)
Tagging node:8.5.0-alpine
Tagging node:8.5-alpine
Tagging node:8-alpine
Tagging node:alpine

$ test/run.sh node:8.5.0-alpine
testing node:8.5.0-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.5.0-onbuild
Building bashbrew/cache:1c9a8fbf13e0ed2736712feb46fa6bddefa1b1042a1902df0098359c136fe846 (node:8.5.0-onbuild)
Tagging node:8.5.0-onbuild
Tagging node:8.5-onbuild
Tagging node:8-onbuild
Tagging node:onbuild

$ test/run.sh node:8.5.0-onbuild
testing node:8.5.0-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:8.5.0-slim
Building bashbrew/cache:c23d38be500615937cd7826692267d68b1d8bdf132f119e3dd36bab627b49f66 (node:8.5.0-slim)
Tagging node:8.5.0-slim
Tagging node:8.5-slim
Tagging node:8-slim
Tagging node:slim

$ test/run.sh node:8.5.0-slim
testing node:8.5.0-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.5.0-stretch
Building bashbrew/cache:a9726c448fd735caf95c27758c4af11802648ce5d19570527a2beb25f710c29f (node:8.5.0-stretch)
Tagging node:8.5.0-stretch
Tagging node:8.5-stretch
Tagging node:8-stretch
Tagging node:stretch

$ test/run.sh node:8.5.0-stretch
testing node:8.5.0-stretch
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.5.0-wheezy
Building bashbrew/cache:37a0c632d4b379389deaeb7fe892cc9365a58579072c477976fe24ddc0c77580 (node:8.5.0-wheezy)
Tagging node:8.5.0-wheezy
Tagging node:8.5-wheezy
Tagging node:8-wheezy
Tagging node:wheezy

$ test/run.sh node:8.5.0-wheezy
testing node:8.5.0-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.11.3
Building bashbrew/cache:f93150c7b22662c920675b555772a9d0a3f0d964487ab01473dfa99333cb9d7d (node:6.11.3)
Tagging node:6.11.3
Tagging node:6.11
Tagging node:6
Tagging node:boron

$ test/run.sh node:6.11.3
testing node:6.11.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.11.3-alpine
Using bashbrew/cache:0349223999aff896f3062b4652bb18622edb40e19736640a1aea0411e4ea3d0f (node:6.11.3-alpine)
Tagging node:6.11.3-alpine
Tagging node:6.11-alpine
Tagging node:6-alpine
Tagging node:boron-alpine

$ test/run.sh node:6.11.3-alpine
testing node:6.11.3-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.11.3-onbuild
Building bashbrew/cache:f77b7da5b370ee19519fd15995c94da6881687008deaf1afcc1c19109ede12cc (node:6.11.3-onbuild)
Tagging node:6.11.3-onbuild
Tagging node:6.11-onbuild
Tagging node:6-onbuild
Tagging node:boron-onbuild

$ test/run.sh node:6.11.3-onbuild
testing node:6.11.3-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:6.11.3-slim
Building bashbrew/cache:780b8135080d8590637ba1e7778cb3bbfe8aa8f10b36e2aadd753f2b55ee0fb2 (node:6.11.3-slim)
Tagging node:6.11.3-slim
Tagging node:6.11-slim
Tagging node:6-slim
Tagging node:boron-slim

$ test/run.sh node:6.11.3-slim
testing node:6.11.3-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.11.3-stretch
Building bashbrew/cache:63a79ab1189e304bdc814f25b46e143b88d37e4be77c0efb8be8d1e36ba7bb89 (node:6.11.3-stretch)
Tagging node:6.11.3-stretch
Tagging node:6.11-stretch
Tagging node:6-stretch
Tagging node:boron-stretch

$ test/run.sh node:6.11.3-stretch
testing node:6.11.3-stretch
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.11.3-wheezy
Building bashbrew/cache:37b85a64269d60d1f9e66ec37b109e95fc9ce97c32c4e4726cc17edd6655d9f6 (node:6.11.3-wheezy)
Tagging node:6.11.3-wheezy
Tagging node:6.11-wheezy
Tagging node:6-wheezy
Tagging node:boron-wheezy

$ test/run.sh node:6.11.3-wheezy
testing node:6.11.3-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.4
Building bashbrew/cache:89b539aeb6371c9f5dcefed1618b71c1f04aa3ba04f6548691c7c3eed90b787e (node:4.8.4)
Tagging node:4.8.4
Tagging node:4.8
Tagging node:4
Tagging node:argon

$ test/run.sh node:4.8.4
testing node:4.8.4
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.4-alpine
Using bashbrew/cache:e6272cd77c7c2bca3cbf32e910a9f28c11b9e697e108300cc83a832be8fb1aa2 (node:4.8.4-alpine)
Tagging node:4.8.4-alpine
Tagging node:4.8-alpine
Tagging node:4-alpine
Tagging node:argon-alpine

$ test/run.sh node:4.8.4-alpine
testing node:4.8.4-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.4-onbuild
Building bashbrew/cache:71750cb8bbd4f18580184510c30298327366c5e9c418734c91f3b6cd9a674fae (node:4.8.4-onbuild)
Tagging node:4.8.4-onbuild
Tagging node:4.8-onbuild
Tagging node:4-onbuild
Tagging node:argon-onbuild

$ test/run.sh node:4.8.4-onbuild
testing node:4.8.4-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:4.8.4-slim
Building bashbrew/cache:c94fb2eba72370d0557c35172ddcc7df57b23df21cc0c1001fe13807ccb09b7c (node:4.8.4-slim)
Tagging node:4.8.4-slim
Tagging node:4.8-slim
Tagging node:4-slim
Tagging node:argon-slim

$ test/run.sh node:4.8.4-slim
testing node:4.8.4-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.4-stretch
Building bashbrew/cache:b5e923b147414805ef80d03ff50135cfa524bf3eaa5ab2ab8538a816a5538bdb (node:4.8.4-stretch)
Tagging node:4.8.4-stretch
Tagging node:4.8-stretch
Tagging node:4-stretch
Tagging node:argon-stretch

$ test/run.sh node:4.8.4-stretch
testing node:4.8.4-stretch
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.4-wheezy
Building bashbrew/cache:826cd48de2cbeacb1cff64c6c347cf9206fea716e77b7c01a90bc565db7f992b (node:4.8.4-wheezy)
Tagging node:4.8.4-wheezy
Tagging node:4.8-wheezy
Tagging node:4-wheezy
Tagging node:argon-wheezy

$ test/run.sh node:4.8.4-wheezy
testing node:4.8.4-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed

```

**Pull request merged**